hr_public_holidays/models/hr_public_holidays.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding:utf-8 -*-
```

Comment:
Modify licence format to suit with the others: see example 
https://github.com/OCA/hr/blob/8.0/hr_expense_invoice/__openerp__.py